source/test_linked_list.cxx

```diff
@@ -0,0 +1,405 @@
+#include "LinkedList.h"
+
+#include <gtest/gtest.h>
```

Comment:
This is much more readable! Looks like you're missing coverage for your copy/move ctors/assign operators. Are you running your library through any sort of coverage tool? That'd be a nice touch.

Reply:
I'm running it through gcov, which says I have 100% coverage.  Maybe I've configured it wrong?

Reply:
Hmm. I wonder where your move assign/ctor are being exercised. Either way, leak in those as described above.